dna-src/zomes/graphql/code/src/identity/mod.rs

```diff
@@ -31,10 +33,10 @@ pub fn get_identity(agent_id: &Address) -> ZomeApiResult<Identity> {
 }
 
 pub fn register_user(name: String, avatar_url: String, hylo_id: String) -> ZomeApiResult<Address> {
-    // don't reregister everyone if it has been done already
+    // only register the test identities if there are none already
     let n_registered = get_people()?.len();
-    if n_registered > 0 {
```

Comment:
This was the offending line. It aborts with a fake success if it detects existing agents. This explains the behavior we were seeing when testing online where a registration would appear to work but not actually

Reply:
Great news! We can now bring on the n3h magic.

Reply:
Does this now means we won't have the Beatles at all once there is more than 1 real agent seen?

Reply:
kinda. The first agent will add the Beatles and the other agents will get them via gossip